src/main/modules/api/cos-api.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+import { LoggerInstance } from 'winston';
+
+export class CosApi {
```

Comment:
Perhaps we should start adding unit tests for new modules?

Reply:
this shouldn't be a module and it shouldn't be cos-api.ts :)

Reply:
modules should just be for express middleware setup